fastestimator/dataset/data/cifair10.py

```diff
@@ -52,9 +52,8 @@ def load_data(root_dir: str = None, image_key: str = "x", label_key: str = "y")
     image_extracted_path = os.path.join(root_dir, 'ciFAIR-10')
 
     if not os.path.exists(image_extracted_path):
-        if not os.path.exists(image_compressed_path):
-            print("Downloading data to {}".format(root_dir))
-            _download_file_from_google_drive('1dqTgqMVvgx_FZNAC7TqzoA0hYX1ttOUq', image_compressed_path)
+        print("Downloading data to {}".format(root_dir))
```

Comment:
why are we removing these checks for if we've already downloaded the zip? in that case we definitely should check for whether we already have a valid zip before re-downloading

Reply:
Currently we are validating before download. Instead of doing it every time we are using google download, we are just moving the if check instead the download, it would check the file and download if the file doesn't exist.